groups/instance/cloud-init/templates/enable_config.tpl

```diff
@@ -749,8 +749,8 @@ write_files:
 
 runcmd:
   - systemctl enable artifactory
-  - sudo mount -t efs -o tls ${efs_dns_name}:/ /var/lib/artifactory
-  - sudo echo "${efs_dns_name}:/ /var/lib/artifactory efs defaults,_netdev,noresvport,tls 0 0" >> /etc/fstab
+  - sudo mount -t efs -o tls,accesspoint=${efs_access_point_id} ${efs_filesystem_id} /var/lib/artifactory
+  - sudo echo "${efs_filesystem_id} /var/lib/artifactory efs _netdev,tls,accesspoint=${efs_access_point_id} 0 0" >> /etc/fstab
```

Comment:
I would suggest a small update here just to ensure the `fstab` entry is good and stable.

Rather than mounting the EFS and writing the string to `fstab`, it would be better to write the string to `fstab` and then execute the `mount -a` command to process the `fstab`.

E.g.:
```
  - sudo echo "${efs_filesystem_id} /var/lib/artifactory efs _netdev,tls,accesspoint=${efs_access_point_id} 0 0" >> /etc/fstab
  - sudo mount -a
```

Reply:
All mounting has been implemented via fstab as required